Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment.ts

```diff
@@ -18,8 +18,8 @@ async function run() {
 		var deployToSlotFlag: boolean = tl.getBoolInput('DeployToSlotFlag');
 		var resourceGroupName: string = tl.getInput('ResourceGroupName');
 		var slotName: string = tl.getInput('SlotName');
-		var package: string = tl.getPathInput('Package');
-		var setParametersFile: string = tl.getPathInput('SetParametersFile');
+		var webDeployPkg: string = trimDoubleQuotes(tl.getPathInput('Package'));
```

Comment:
You can't do this.  Double quote is a valid char in paths on unix.  Also not clear what the motivation is.  There's no requirement to double quote a single value.  Paths with spaces survive without needing the double quote.  And the picker won't double quote.